PR for Issues #26,  #22, and ticket --> Flutter SDK - WASM compatibility improvements

muteeb-shahid1 opened the pull request:
# Explain your changes

Flutter SDK - package not working
1. A user reported an issue that package was not workin gon IOS, It was creating the issue related to the webview on IOS. I have fixed that issue, run the SDK on IOS and its working fine. 

2. There is ticket to replace the hive with hive_ce, and html with web. So, I have replaced the related packages, and test the things, and all working fine.

Issue --> Here is an other issue Flutter SDK - IsAuthenticate return false when logout and than login
3. User reported the issue that when user logout and try to relogin, sdk through an isseu related to the authenitcation, I have fixed the issue, and test it completely on IOS, as user's mentioned platform is IOS.

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

DanielRivers commented on CLIENT_DELIVERY_SUMMARY.md:
This appears to be AI generated summary md file, please can it be removed?

DanielRivers commented:
Thank you for submitting this PR, I will pass onto the relevant people to get it reviewed

muteeb-shahid1 commented:
Here is the details of the changes

1. iOS WebView Build Failure

Code Change: Updated dependencies in pubspec.yaml and migrated the WebView library. Verified successful integration of the Kinde Flutter SDK by testing in the sample project.

2. Flutter SDK – Compatibility Improvements

Code Change: Replaced hive with hive_ce and migrated associated functionalities. Replaced html with web along with related functions.

3. Android NDK Build Conflicts

Code Change: Updated Android build configuration by upgrading the NDK version to 27.0.12077973.

4. Authentication State Bug (isAuthenticated returning false)

Code Change: Added _instance = null; inside the _commonLogoutCleanup() method (lib/src/kinde_flutter_sdk.dart:275).
